### ISSUE TITLE
Creating an AdoptOpenJDK Specific Mailing List

### ISSUE THREAD

**Issue by SeanC1083** [AdoptOpenJDK/openjdk-website#388]
This Git Issue was created following a conversation with Martijn (@karianna) about both the lack of a dedicated, unique Mailing List for AdoptOpenJDK, and the goal of improving the accessibility and the need for long-term content that users want to see. Also, Mailing List is one of the preferred ways to get the type of information that users expect from Mailing Lists.
